pubspec.yaml

```diff
@@ -69,18 +69,15 @@ dependencies:
   webfeed: ^0.7.0
 
 dependency_overrides:
-  firebase_analytics_web:
-    git: 
-      url: https://github.com/durduman/flutterfire.git
-      path: packages/firebase_analytics/firebase_analytics_web
-      ref: firebase_analytics_web-0.4.0+16-fix-for-web
+  web3dart: ^2.4.1
```

Comment:
The latest version of `web3dart` is causing `build_runner` issues.

On my AX-637 branch I'm using an override:
```
  web3dart:
    git:
      url: https://github.com/narcodico/web3dart.git
      ref: fix/build-runner
```
from a fix available here: https://github.com/narcodico/web3dart/tree/fix/build-runner

Try running `flutter pub run build_runner build --delete-conflicting-outputs` from the root of the app, with your existing fix to see if it's working or if the mentioned fix is still needed.

Reply:
Yeah it breaking for me when I try the build runner. We would definitely have to use either @durduman git fix or yours. The question is which one is safer?

Reply:
His fix is for some other windows specific issue.